web/public/locales/hi.json

```diff
@@ -49,8 +49,6 @@
     "renewable": "अक्षय",
     "electricityproduction": "बिजली उत्पादन",
     "electricityconsumption": "बिजली खर्च",
-    "bysource": "श्रोत से",
-    "averagebysource": "श्रोतों का औसत",
```

Comment:
Also don't feel comfortable trusting the automated translations for these, as I cannot verify as easily without Latin alphabet.

Reply:
Agreed